### DISCUSSION TITLE
How to make the cell background white when using color_default='black on white'

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2135] [Q&A] [answered]
I'm back on visidata! :-)

I have set 

    vd.options.color_default = 'black on white'

but the background of normal cells is **grey** not white. How can I set the background to white?

PS: Unfortunately on the computer in question I'm using still version 2.8 which I cannot easily change (no internet - machine is air-gapped). So using `theme-input` is not possible.

**Comment by saulpw** [comment]
Hey @halloleo, welcome back!  Try `black on 15`; the standard `white` is color 8 which is a bit dimmer than the brightest white.  You can also play with colors 232-255 for more gradations.

**Reply by halloleo** [reply]
Thanks! `black on 15` is still pretty grey... Is there a way to give it RGB (to make really sure the system knows what I mean...)

**Reply by saulpw** [reply, marked as answer]
Unfortunately not really.  There are some terminals that support RGB, but curses does not.  Look into the color palette in your terminal settings, you should be able to adjust the colors there.  One of 15 or 231 or 255 should be the brightest white.

**Reply by halloleo** [reply]
I was fearing this. I use `vd` inside a vterm in Emacs, so lots of layers of terminal translation...

I guess I will need to bite the bullet and _understand_ what translation the different layers do...

**Reply by halloleo** [reply]
Actually, I got it working your suggestion. The line

    vd.options.color_default = 'black on 255'

makes the background completely white. 😃 

Thank you! 🙏
